djangocms_form_builder/cms_plugins/form_plugins.py

```diff
@@ -69,7 +70,6 @@ class CharFieldPlugin(mixin_factory("CharField"), FormElementPlugin):
     name = _("Text")
     model = models.CharField
     form = forms.CharFieldForm
-    settings_fields = (("min_length", "max_length"),)
```

Comment:
Why do you remove `min_length` and `max_length`? aren't they potentially needed for form validation?